<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1051</width>
    <height>688</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1051</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-3</y>
      <width>1051</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../view_img/fondo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-3</y>
      <width>1051</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:00715909, stop:0 rgba(0, 0, 0, 35), stop:0.375 rgba(0, 0, 0, 50), stop:0.835227 rgba(0, 0, 0, 75));

border: 3px solid white; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,35);
border-radius:13px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_listar_vent">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(241, 241, 241);
}

QPushButton:pressed {
	background-color: rgb(223, 223, 223);
}
</string>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cancelar_pedcompr">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: black;
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(255, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(166, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_nuevo_vent">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(241, 241, 241);
}

QPushButton:pressed {
	background-color: rgb(223, 223, 223);
}
</string>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
   <widget class="QWidget" name="window_1_compr" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>94</y>
      <width>1051</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>1011</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,35);
border-radius:13px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:00715909, stop:0 rgba(0, 0, 0, 20), stop:0.375 rgba(0, 0, 0, 50), stop:0.835227 rgba(0, 0, 0, 75));
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>24</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../view_img/METROSPLASH.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Generar Pedido de Compra</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="GB_2_CLIENT">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>431</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: black;
}

QGroupBox{
color: white
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QComboBox" name="cb_proveedor">
      <property name="geometry">
       <rect>
        <x>141</x>
        <y>26</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
    border: 1px solid #6c6c6c;
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QComboBox::drop-down {
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;    
}
QComboBox::down-arrow {
	border-image: url(:/Imagenes/203484.png);
}

</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Proveedor:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GB_2_CLIENT_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>431</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: black;
}

QGroupBox{
color: white
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QComboBox" name="cb_product_ped">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
    border: 1px solid #6c6c6c;
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QComboBox::drop-down {
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;    
}
QComboBox::down-arrow {
	border-image: url(:/Imagenes/203484.png);
}

</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>24</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Producto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>84</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Cantidad:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cant_ped">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>78</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(0, 0);
color:rgba(255, 255, 255, 230);
padding-bottom:7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_add_pr_ped">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(0, 255, 0);
}

QPushButton:pressed {
	background-color: rgb(0, 217, 0);
}</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GB_2_CLIENT_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>130</y>
       <width>521</width>
       <height>271</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: black;
}

QGroupBox{
color: white
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QTableWidget" name="tw_showPed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>501</width>
        <height>241</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    border: 1px solid #d6d9dc;
    background-color: #f9fafb;
    gridline-color: #e6e9ec;
    alternate-background-color: #f1f3f4;
}

QHeaderView::section {
    background-color: #e6e9ec;
    padding: 4px;
    border: 1px solid #d6d9dc;
    font-weight: bold;
    color: #333333;
}

QTableWidget::item {
    border: 1px solid #d6d9dc;
    padding: 5px;
}

QTableWidget::item:selected {
    background-color: #d6d9dc;
    color: #333333;
}</string>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Producto</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Medida</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Precio</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cantidad</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="bt_guardar_ped">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>448</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(241, 241, 241);
}

QPushButton:pressed {
	background-color: rgb(223, 223, 223);
}
</string>
     </property>
     <property name="text">
      <string>Guardar
Pedido</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_clear_ped">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>448</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f6f7fa, stop:1 #dadbde);
	border: 2px solid #444444;
    text-align: center; 
    text-decoration: none; 
    border-radius: 12px;
}

QPushButton:hover {
	background-color: rgb(241, 241, 241);
}

QPushButton:pressed {
	background-color: rgb(223, 223, 223);
}
</string>
     </property>
     <property name="text">
      <string>Limpiar
Registro</string>
     </property>
    </widget>
    <widget class="QLabel" name="warning_ped">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>403</y>
       <width>631</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>label_10</zorder>
    <zorder>GB_2_CLIENT_3</zorder>
    <zorder>GB_2_CLIENT</zorder>
    <zorder>bt_guardar_ped</zorder>
    <zorder>bt_clear_ped</zorder>
    <zorder>GB_2_CLIENT_4</zorder>
    <zorder>label_4</zorder>
    <zorder>label_7</zorder>
    <zorder>label_6</zorder>
    <zorder>warning_ped</zorder>
   </widget>
   <widget class="QWidget" name="window_2_lst" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>93</y>
      <width>1051</width>
      <height>591</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>12</y>
       <width>1011</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,35);
border-radius:13px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="fondo3">
     <property name="geometry">
      <rect>
       <x>835</x>
       <y>27</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../view_img/METROSPLASH.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>32</y>
       <width>551</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,35);
border-radius:13px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bol_fact">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>42</y>
       <width>611</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Pedidos de Compra Registrados</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="GB_2_CLIENT_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>951</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
    color: black;
}

QGroupBox{
color: white
}
</string>
     </property>
     <property name="title">
      <string>Selecciona para ver mas detalles</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <widget class="QTableWidget" name="tw_PED_COMPR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>41</y>
        <width>911</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
    border: 1px solid #d6d9dc;
    background-color: #f9fafb;
    gridline-color: #e6e9ec;
    alternate-background-color: #f1f3f4;
}

QHeaderView::section {
    background-color: #e6e9ec;
    padding: 4px;
    border: 1px solid #d6d9dc;
    font-weight: bold;
    color: #333333;
}

QTableWidget::item {
    border: 1px solid #d6d9dc;
    padding: 5px;
}

QTableWidget::item:selected {
    background-color: #d6d9dc;
    color: #333333;
}</string>
      </property>
      <column>
       <property name="text">
        <string>IdCompra</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Auxiliar Asistente</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Codigo Factura</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Proveedor</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RUC</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Total por Compra</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estado</string>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_9</zorder>
   <zorder>bt_cancelar_pedcompr</zorder>
   <zorder>bt_nuevo_vent</zorder>
   <zorder>bt_listar_vent</zorder>
   <zorder>window_1_compr</zorder>
   <zorder>window_2_lst</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../view_img/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
